src/app/game/services/game-state-service.ts

```diff
@@ -0,0 +1,102 @@
+import { GAMESTATE } from '@/app/shared/models/interfaces';
+import { Injectable } from '@angular/core';
+import { BehaviorSubject } from 'rxjs';
+import { GameConfigService } from './game-config.service';
+
+enum GAMEMODE {
+  singleplayer = 'SINGLEPLAYER',
+  multiplayer = 'MULTIPLAYER',
+  notSet = 'NOTSET',
+}

```

Comment:
Have this here or in interfaces?

Reply:
Holdes her frem til den eventuelt blir brukt i noen andre klasser.